DebugKit.podspec

```diff
@@ -8,7 +8,7 @@
 
 Pod::Spec.new do |s|
 s.name             = 'DebugKit'
-s.version          = '1.0.0'
+s.version          = '1.0.1'
```

Comment:
Is the podspec the only place we need to update the version number or do we also need to update it in the Xcode project or Package.swift file?

Reply:
SPM uses the tag on the repo. Nothing uses the Xcode project but I went back and updated it for consistency.